canonical-web-and-design/maas-ui
# WIP: Improve machine list sorting and grouping.

## #700 prerequisite

## Done
- Copied over some grouping code from the angular machine list, which orders groups in specific ways (mostly with status)
- Added a GroupSelect component, which is simply a select that lets you choose the grouping

## QA
- Check that the machine list is sorted by status by default, where failed machines show up first
- Check that you can change the grouping using the right select
- Check that you can sort within groups by clicking the table headers
